templates/repo/graph.tmpl

```diff
@@ -8,7 +8,6 @@
 				<div class="ui icon buttons tiny color-buttons">
 					<div class="ui multiple selection search dropdown" id="flow-select-refs-dropdown">
 						<input type="hidden" name="flow">
-						<i class="dropdown icon"></i>
```

Comment:
What happened to this icon?

Reply:
It was previously hidden via CSS so was never visible. The field still works without it as expected (git graph branch selector).